packages/opentelemetry-cloud-monitoring-exporter/package.json

```
@@ -46,9 +46,9 @@
   "devDependencies": {
     "@google-cloud/ops-js-scripts": "^1.0.0",
     "@opentelemetry/api": "1.9.0",
-    "@opentelemetry/core": "1.26.0",
-    "@opentelemetry/resources": "1.26.0",
-    "@opentelemetry/sdk-metrics": "1.26.0",
+    "@opentelemetry/core": "2.0.0",
+    "@opentelemetry/resources": "2.0.0",
+    "@opentelemetry/sdk-metrics": "2.0.0",
```

```suggestion
    "@opentelemetry/core": "^2.0.0",
    "@opentelemetry/resources": "^2.0.0",
    "@opentelemetry/sdk-metrics": "^2.0.0",
```

What is a reason to pin these dev dependencies to the specific version?

---

We follow the Renovate recommendations https://docs.renovatebot.com/dependency-pinning/#why-pin-dependencies